creator-node/compose/docker-compose.yml

```diff
@@ -50,7 +50,7 @@ services:
     networks:
       - audius_dev
   creator-node-redis:
-    image: redis:5.0.4
+    image: redis:6.2.7
```

Comment:
why not direct to redis 7?

Reply:
I thought it'd be better to do one major version at a time - redis 6 didn't have a ton of changes, but redis 7 has quite a few breaking changes. I guess I could bump further to 7 though if we want to just test on staging a bit longer - wdyt? and yeah no code change needed for this one as far as I can tell

Reply:
ya agree on separating, this is so simple we can ship low risk and get whatever incremental benefits it provides